x-pack/plugins/vis_type_timeseries_enhanced/README.md

```diff
@@ -0,0 +1,10 @@
+# vis_type_timeseries_enhanced
+
+The `vis_type_timeseries_enhanced` plugin is the x-pack counterpart to the OSS `vis_type_timeseries` plugin.
+
+It exists to provide Elastic-licensed services, or parts of services, which
+enhance existing OSS functionality from `vis_type_timeseries_enhanced`.
```

Comment:
Maybe here you mean `vis_type_timeseries`?

Reply:
thank you